charts/devnet/templates/chains/cosmos/service.yaml

```diff
@@ -9,6 +9,10 @@ metadata:
   name: {{ $chain.name }}-genesis
   labels:
     app.kubernetes.io/name: {{ $chain.name }}-genesis
+  annotations:
+    cloud.google.com/backend-config: '{"ports": {
+    "rest": "rest-hc-config"
```

Comment:
Where do you take values like `rest-hc-config` from?

Reply:
rest-hc-config and explorer-hc-config come from test-backendconfig.yaml in our starship-config repo.

This wouldn't be suitable for upstreaming. We could hide it in another `annotations` field in the yaml.